### PR TITLE
AC Automated Tests - LRAC-13504 | master 

### PR DESCRIPTION
**Pull request by manuferreira**
[PR 1:](https://github.com/liferay-ac/liferay-portal/pull/404)
https://issues.liferay.com/browse/LRAC-13506
https://issues.liferay.com/browse/LRAC-13507
https://issues.liferay.com/browse/LRAC-13508
https://issues.liferay.com/browse/LRAC-13509
https://issues.liferay.com/browse/LRAC-13489

[PR 2:](https://github.com/liferay-ac/liferay-portal/pull/405)
https://issues.liferay.com/browse/LRAC-13504
https://issues.liferay.com/browse/LRAC-13505

[PR 3:](https://github.com/liferay-ac/liferay-portal/pull/406)
https://issues.liferay.com/browse/LRAC-13510

PR 4:
https://issues.liferay.com/browse/LRAC-13495

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase [hunk 1]
```diff
@@ -1053,4 +1053,81 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-12919 | Automation ID: LRAC-13495 | Test Summary: Check that you can access data from all existing segments"
+    @priority = 4
+    test RequestToAccessDataForAllSegments {
+        var segmentsTitleList = "managers,Annual Revenue > $500K,Dynamic Segment Test";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase]: Hey @manuferreira, I think validating those segments that are created by the Demo property can be a bit dangerous. Currently, we are not able to delete properties in `tearDown`, but when this functionality comes back, if some other previous test deletes the property and then your test will fail.

I think you could create another segment and verify that the segments created in your test is showing up in the API response

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase [hunk 2]
```diff
@@ -24,6 +24,8 @@ definition {
 	tearDown {
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		ACUtils.tearDownDXP();
+
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase:28]: Hey @manuferreira, could you add the `tearDown` macro from AC here following how it was used in this [other file](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase#L28) (same comment and commented macro in the code)?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase (hunk 1 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase]: Done :heavy_check_mark:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase (hunk 2 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase:30]: Done :heavy_check_mark:

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
Very good @manuferreira, muito obrigado viu. LGTM 🎉 🕵🏽‍♀️

**Review by joshitagaki — APPROVED** [review]
Great job! Thank you guys for working on these tests

**Review by nkkwon — APPROVED** [review]
Thank you for working on this! LGTM! :+1:

**Comment by manuferreira** [comment]
Thanks for reviewing it!

**Comment by manuferreira** [comment]
ci:forward

**Pull request closed without merging**